activerecord/test/cases/query_logs_test.rb

```diff
@@ -185,7 +185,7 @@ def test_empty_comments_are_not_added
 
   def test_sql_commenter_format
     ActiveRecord::QueryLogs.tags_formatter = :sqlcommenter
-    ActiveRecord::QueryLogs.tags = [:application, {}]
+    ActiveRecord::QueryLogs.tags = [:application]
```

Comment:
With this change we could remove this line, but I think it's worth setting the value we're testing for